applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -1300,20 +1228,69 @@ int le_audio_pa_sync_set(struct bt_le_per_adv_sync *pa_sync, uint32_t broadcast_
 	return -ENOTSUP;
 }
 
-int le_audio_play_pause(void)
+int le_audio_play(void)
 {
-	int ret;
+	int ret_left = 0;
+	int ret_right = 0;
 
-	if (IS_ENABLED(CONFIG_STREAM_BIDIRECTIONAL)) {
-		LOG_WRN("Play/pause not supported for bidirectional mode");
-	} else {
-		ret = ble_mcs_play_pause(NULL);
-		if (ret) {
-			LOG_WRN("Failed to change streaming state");
-			return ret;
+	if (ep_state_check(headsets[AUDIO_CH_L].sink_stream.ep, BT_BAP_EP_STATE_QOS_CONFIGURED)) {
+		ret_left = bt_bap_stream_enable(&headsets[AUDIO_CH_L].sink_stream,
+						lc3_preset_sink.codec.meta,
+						lc3_preset_sink.codec.meta_count);
+
+		if (ret_left) {
+			LOG_WRN("Failed to enable left stream: %d", ret_left);
+		}
+	}
+
+	if (ep_state_check(headsets[AUDIO_CH_R].sink_stream.ep, BT_BAP_EP_STATE_QOS_CONFIGURED)) {
```

Comment:
If `ret_left != 0`, does it make sense to handle the right one at all? 

The error you return is EIO if any of them fails, but it will be unclear to the caller which or if both failed.

Reply:
Why wouldn't you handle the two devices separately. If the left device disconnected right in the middle of this it could be failing, but we should still be able to pause the right. 

They do print individually first though.

Reply:
> They do print individually first though.

Sure, but that is only useful if there's a person looking at the logs, and not something an application can use for anything :) 

> Why wouldn't you handle the two devices separately. If the left device disconnected right in the middle of this it could be failing, but we should still be able to pause the right.

This assumes that left and right streams are on two different devices. Is this only intended for TWS, or will it also support stereo headsets?

Reply:
> This assumes that left and right streams are on two different devices. Is this only intended for TWS, or will it also support stereo headsets?

Shouldn't matter what devices they are. It only assumes that they are two different streams.

Reply:
> Shouldn't matter what devices they are. It only assumes that they are two different streams.

Arguably, yes. The question is: If I have a stereo headset, do I prefer to get no audio, or audio in just one side? 

That's probably subjective, and I'll leave that to you (none of the LE Audio specs declare any "right" behavior for this IIRC). 

In any case, it will still be unclear to the caller of this function whether one or two streams failed to be started, or which one failed, right?

Reply:
No, left...